src/main.rs

```diff
@@ -43,6 +43,12 @@ fn main() {
                 .short("c")
                 .long("normalize-case"),
         )
+        .arg(
+            Arg::with_name("enforce-nullable")
```

Comment:
I would prefer `force-nullable` since `enforce` implies there is some extant thing being enforced when it is the option itself that is causing the behavior change.